### PR TITLE
Synth categories + additional grantsDAO content

### PR DESCRIPTION
**Pull request by garthtravers**

### CODE REVIEW

File: content/governance.md [hunk 1]
```diff
@@ -16,9 +16,13 @@ SCCPs are similar to SIPs, but concern modifications to system configuration val
 
 SCCPs live in the same repository and website as SIPs do, but have a [slightly different specification](https://github.com/Synthetixio/SIPs/blob/master/SCCP/sccp-1.md).
 
-## Synthetix GrantsDAO
+## Synthetix grantsDAO
 
-The Synthetix GrantsDAO was set up to fund community grant proposals for such purposes as infrastructure and promotional resources. It consists of 3 community members and 2 team members. Open grants can be [viewed](https://snxgrants.io/) by the public, and new grant proposals can be [submitted](https://github.com/Synthetixio/snx-grants-dao/blob/master/proposals/proposal-template.md).
+The Synthetix grantsDAO was set up to fund community grant proposals for such purposes as infrastructure and promotional resources. The DAO consists of 3 community members and 2 core contributors, and there is an interface available for the public [here](https://snxgrants.io/).
```
Comment (justinjmoses) [inline content/governance.md:21]: [np] It's generally best practice to provide hyperlinks with more informative text than "here". How about something like: 

```markdown
...there is an interface available for the public at [snxgrants.io](https://snxgrants.io/). 
```

So basically strip our the protocol and slashes where possible.

File: content/governance.md [hunk 2]
```diff
@@ -16,9 +16,13 @@ SCCPs are similar to SIPs, but concern modifications to system configuration val
...
+## Synthetix grantsDAO
 
-The Synthetix GrantsDAO was set up to fund community grant proposals for such purposes as infrastructure and promotional resources. It consists of 3 community members and 2 team members. Open grants can be [viewed](https://snxgrants.io/) by the public, and new grant proposals can be [submitted](https://github.com/Synthetixio/snx-grants-dao/blob/master/proposals/proposal-template.md).
+The Synthetix grantsDAO was set up to fund community grant proposals for such purposes as infrastructure and promotional resources. The DAO consists of 3 community members and 2 core contributors, and there is an interface available for the public [here](https://snxgrants.io/). 
+
+Grants are intended for individuals or teams who support Synthetix and are willing to contribute to the project in some way. There is a maximum of 50,000 SNX per grant, though most are expected to be much smaller projects than this, and proposals can be submitted [here](https://github.com/Synthetixio/snx-grants-dao/blob/master/proposals/proposal-template.md). The intention behind the grantsDAO is to reward people who already hold SNX and help further the ecosystem in some way.
```
Comment (justinjmoses) [inline content/governance.md:23]: For this one, maybe:

```markdown
...can be submitted as a PR on the [snx-grants-dao repo](...)
```

Comment (justinjmoses) [inline content/governance.md:23]: Also, in markdown you can surround numbers and code with backticks (as in: \`). So I'd suggest it for `50,000 SNX`

File: content/governance.md [hunk 3]
```diff
@@ -16,9 +16,13 @@ SCCPs are similar to SIPs, but concern modifications to system configuration val
...
+The Synthetix grantsDAO was set up to fund community grant proposals for such purposes as infrastructure and promotional resources. The DAO consists of 3 community members and 2 core contributors, and there is an interface available for the public [here](https://snxgrants.io/). 
+
+Grants are intended for individuals or teams who support Synthetix and are willing to contribute to the project in some way. There is a maximum of 50,000 SNX per grant, though most are expected to be much smaller projects than this, and proposals can be submitted [here](https://github.com/Synthetixio/snx-grants-dao/blob/master/proposals/proposal-template.md). The intention behind the grantsDAO is to reward people who already hold SNX and help further the ecosystem in some way. 
+
+It's recommended that before you go to the trouble of submitting a proposal to the grantsDAO, that you either gauge community sentiment around the utility of your proposed addition to the ecosystem, or contact a grantsDAO member to discuss it directly. This can be done through the [Synthetix Discord](https://discord.com/invite/AEdUHzt) in the #grants-dao channel.
```
Comment (justinjmoses) [inline content/governance.md:25]: 👍 to this link

File: content/tokens/categories.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Let's say iBTC was added into the system when sBTC was at $5000. That would mean iBTC's entry point would be $5000, and its value would inversely track that of sBTC, so if the price of sBTC drops to $4900 then iBTC would be at $5100, and vice versa.  
+
+Let's also say that iBTC's upper limit is $7500 and lower limit is $2500 (these are programmed when it's added to the system). These represents the points at which this iSynth gets frozen. If sBTC reaches $2500, then iBTC reaches its upper limit of $7500 and is frozen at that value. Any further fluctuations in the value of sBTC will not be reflected in the value of iBTC. The Synthetix core contributors will then endeavour to redeploy the frozen iSynth as soon as possible. This involves 'purging' everyone who holds the frozen iSynth into sUSD, after which a new iBTC is then deployed with a new entry point and new limits.  
+
+**iSynth leverage**
+
+The primary reason iSynths need upper and lower limits is because the further an iSynth gets from its entry point, the more effective leverage there is for each movement. 
+
+**An example of how iSynth leverage works**
```
Comment (justinjmoses) [inline content/tokens/categories.md:35]: Do you reckon @zoe-archer could mock up some graphic here to illustrate some of this math?

Comment (justinjmoses) [reply content/tokens/categories.md:35]: Additionally, it's probably worth taking the math from this section: https://docs.synthetix.io/contracts/exchangerates/#rateorinverted and copying it over here as well

File: mkdocs.yml [hunk 5]
```diff
@@ -35,7 +35,8 @@ nav:
   - Releases:
       - '': releases.md
   - Tokens:
-      - '': tokens.md
+      - Home: tokens/index.md
+      - Categories: tokens/categories.md
```
Comment (justinjmoses) [inline mkdocs.yml:39]: Do you think perhaps we have the category information on the `index` page and then the list of tokens on a separate page? Or even have tokens homepage explain the various tokens in the system - `SNX` the collateral token and all the synths, and then each synth can have a category `crypto`, `forex`, `commodity`, `equities` (this should really be called `equity` in `synths.json` to be consistent @jacko125), and then features like `inverse` or `index` - and these features can be combined - for example `iDEFI` is both `inverse` and `index`. 

It's a bit confusing though so perhaps @zoe-archer might have thoughts here how to help people understand the differences?

Comment (garthtravers) [reply mkdocs.yml:39]: @justinjmoses and @zoe-archer want to take a look at the new changes I made in regards to this feedback? https://github.com/garthtravers/synthetix-docs/tree/master/content/tokens

I like the idea of adding a category to the 'list' section (that page should probably be renamed — 'contracts' maybe?)

**Comment by justinjmoses** [comment]
Closed in favor of #22

**Pull request closed without merging**